### ISSUE TITLE
Trying replicate the same solution A100 

### ISSUE THREAD

**Issue by NiteeshReddyB** [wsxqaza12/LLMAvatarTalk-NVIDIA-RIVA-Audio2Face-Langchain#1]
Do i need to install omini verse if so how to install and run it on Linux

**Comment by Retroplaya135** [comment]
Yes, their architecture also shows how Nvidia Omniverse is used for Audio2Face. See this guide below to install it:

https://docs.omniverse.nvidia.com/launcher/latest/it-managed-launcher/install_guide_linux.html
